src/library/basetracktablemodel.cpp

```diff
@@ -463,6 +467,40 @@ bool BaseTrackTableModel::setData(
     return setTrackValueForColumn(pTrack, column, value, role);
 }
 
+QVariant BaseTrackTableModel::composeCoverArtToolTipHtml(
+        const QModelIndex& index) const {
+    // Determine height of the cover art image depending on the screen size
+    unsigned int absHeightOfCoverartTooltip;
+    const QScreen* primaryScreen = getPrimaryScreen();
+    if (primaryScreen) {
+        absHeightOfCoverartTooltip = static_cast<int>(
+                primaryScreen->availableGeometry().height() *
+                kRelHeightOfCoverartTooltip);
+    } else {
+        qWarning() << "Assuming screen height of 800px.";
+        absHeightOfCoverartTooltip = 800 * kRelHeightOfCoverartTooltip;
+    }
+    // Get image from cover art cache
+    CoverArtCache* pCache = CoverArtCache::instance();
+    QPixmap pixmap = QPixmap(absHeightOfCoverartTooltip,
+            absHeightOfCoverartTooltip); // Height also used as default for the width, in assumption that covers are squares
+    pixmap = pCache->tryLoadCover(this,
+            getCoverInfo(index),
+            absHeightOfCoverartTooltip,
+            CoverArtCache::Loading::NoSignal);
+    if (pixmap.isNull()) {
+        // Cache miss -> Don't show a tooltip
+        return kEmptyString;
```

Comment:
Why not simply QVariant()?

Reply:
Changed! As far as I could see, it behaves the same.